WordPress/src/main/res/values/strings.xml

```diff
@@ -7,7 +7,7 @@
     <string name="xmlrpc_error">Couldn\'t connect. Enter the full path to xmlrpc.php on your site and try again.</string>
     <string name="no_network_title">No network available</string>
     <string name="no_network_message">There is no network available</string>
-    <string name="sign_out_confirm">Sign out?</string>
+    <string name="sign_out_confirm">Sign out of WordPress.com?</string>
```

Comment:
Should this be updated to match the wording presented on the Me tab (Disconnect)?